blueoil/cmd/init.py

```diff
@@ -18,7 +18,8 @@
 import re
 import shutil
 
-import whaaaaat
+import inquirer
+from Cython.Compiler.TypeSlots import inquiry
```

Comment:
This import is necessary?

Reply:
Thank you for point out.
Maybe it came from autocompleted, I removed it.